landoapi/models/secapproval.py

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Models related to the sec-approval process.
+
+See See https://wiki.mozilla.org/Security/Bug_Approval_Process.
+"""
+
+from sqlalchemy.dialects.postgresql.json import JSONB
+
+from landoapi.phabricator import PhabricatorClient
+from landoapi.storage import db
+
+
+class SecApprovalRequest(db.Model):
+    """Represents an event that added a sec-approval comment to a revision."""
+
+    __tablename__ = "secapproval_requests"
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    # The revision ID that this event applies to.
+    revision_id = db.Column(db.Integer, nullable=False)
+
+    # The active diff PHID when sec-approval was requested.
+    diff_phid = db.Column(db.String(128), nullable=False)
```

Comment:
Where are you getting the `128` from? From what I can tell Phabricator stores all phids in `VARBINARY(64)` mysql columns, meaning at most 64 characters in any character encoding that would be sanely used.

If we're not too concerned about throwing errors if some limit is passed, we should probably just use `TEXT` here. Take a read of https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_varchar.28n.29_by_default

and from https://www.postgresql.org/docs/9.1/datatype-character.html

> Tip: There is no performance difference among these three types, apart from increased storage space when using the blank-padded type, and a few extra CPU cycles to check the length when storing into a length-constrained column. While character(n) has performance advantages in some other database systems, there is no such advantage in PostgreSQL; in fact character(n) is usually the slowest of the three because of its additional storage costs. In most situations text or character varying should be used instead.